content/docs/operating/security.md

```diff
@@ -14,7 +14,8 @@ attack vectors that some configurations may enable.
 
 As with any complex systems it is not possible to guarantee that there are no
 bugs. If you find a security bug, please file it in the issue tracker of the
-relevant component.
+relevant component. If you prefer to report privately, please do so to the
+maintainers listed in the MAINTAINERS.md of the relevant repository.
```

Comment:
Do we need to have this layer of indirection? I would prefer to have this go to -team or, potentially, security@p.i which then forwards to -team (for now)

Reply:
We've had one ever, I don't think there's a need to optimise yet.

Reply:
See email thread on -team. if we must externalize this, I would rather be consistent from day 1 as we will never ever be able to retire any address / mechanism we put forth.

Reply:
I don't see a problem with changing this in future, as long as it's easy to find it in our current docs.

I don't think we even have the capability to set up email forwards currently.

Reply:
I think we need to change this soon, but am fine leaving as is for now.